docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -93,6 +97,36 @@ To add a tag, follow these steps:
 1. Select a value from the **Select value** drop-down. This field is populated based upon the tag.
 1. Optional: Select **+** to add an additional tag.
 
+## Optional: Use Aggregate by
+
+{{% admonition type="warning" %}}
+**Aggregate by** is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.
+[Enable the `metricsSummary` feature toggle](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
+{{% /admonition %}}
+
+Using **Aggregate by**, you can calculate RED metrics (span count, erroring span count, and latency information) for `kind=server` spans recently sent to Tempo, grouped by one or more attributes.
+
+Whatever is selected in the **Aggregate by** list, you'll see every combination of present in your data.
+Each aggregate value, for example `intrinsic`:`name`, has a corresponding column in the results table.
+The RED metrics are calculated for every endpoint and user agent combinations found in your data.
```

Comment:
What about this? 

```suggestion
When you use  **Aggregate by**,  the selections you make determine how the information is reported in the Table. Every combination that matches selections in your data is listed in the table. 
Each aggregate value, for example `intrinsic`:`name`, has a corresponding column in the results table.

For example, **names** matching `GET /:endpoint` with a **span.http.user_agent** of `k6/0.46` appeared in 31,466 spans. Instead of being listed by traces and associated spans, the query results are grouped by the the selections in **Aggregate by**.

The RED metrics are calculated for every endpoint and user agent combinations found in your data.
```